### PR TITLE
React capstone project - Metrics webapp

### PR DESCRIPTION
**Pull request by suzi216**
## React capstone project -Crypto Metrics App

- The project is a single-page application (SPA) built with React and Redux
- Choosing the crypto API to be base of the application.
- The data retrieved from the API should be stored in the Redux store.
- Every page (the main page and the pages for each item) should have a unique route within the SPA.
- Prepare routes and navigation.
- Style your components to match the design provided.
- The project should be deployed and accessible online.
- Recording a description video for the project.
- Make sure that there are no linter errors.
- Create  testing for pure functions in the Redux code
- Make sure that you used correct gitflow.
- You should follow these design guidelines.

### CODE REVIEW

**Review by uwadonat — CHANGES_REQUESTED** [review]
## Hi @Suzi216,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
- Your commit messages are very descriptive ✔️
- Your live demo works fine ✔️
- Well organized and clean project ✔️
- No linter errors ✔️
almoo


## Required Changes ♻️
- [ ] Please make sure your video contains a highlights about a piece of the code that you find interesting or proud of making.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100%.
Please, remember to tag me @uwadonat in your question so I can receive the notification.

------
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Covid Metrics
+# Crypto Market Metrics
```
Comment (FrederickMih) [inline README.md:3]: - [**Optional**] Hi @Suzi216, based on the requirement of this project, the README.md file should contain an "Acknowledgements" section or similar that gives appropriate credit to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/).

**Review by FrederickMih — CHANGES_REQUESTED** [review]
Hi @Suzi216, 

Great work on making the changes 👏 
You've done well implementing some of the requested changes, but there are still some which aren't been addressed yet.

## Highlights :green_circle: 

- No Linter error :heavy_check_mark: 

## Suggested changes

- [ ] The previous reviewer requested you to open up your code in your code editor and highlight to us the piece of code you are really proud of or found interesting. Kindly show to us the piece of code.

_Check the comments under the review._

## Optional changes

-  To keep our codebase clean let's remove all the boilerplate that was generated by CRA(unused images, and icons in your root and public directory).  



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Review by rustam-tolipov — APPROVED** [review]
## Hello @Suzi216 👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations! 🎉 🎉 🎉 🎉 🎉 

No Linter errors 💯

Cheers and Happy coding!👏👏👏

Developer-512px-10

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**